src/main/resources/liquibase/changelogs/v3_8_0/schema.yml

```
@@ -20,3 +20,31 @@ databaseChangeLog:
                   type: nclob
                   constraints:
                     nullable: true
+
+        - addColumn:
+            tableName: ${gravitee_prefix}pages
+            columns:
+              - column:
+                  name: excluded_access_controls
+                  type: boolean
+
+        - createTable:
+            tableName: ${gravitee_prefix}page_acl
+            columns:
+              - column: {name: page_id, type: nvarchar(64), constraints: { nullable: false } }
+              - column: {name: reference_id, type: nvarchar(64), constraints: { nullable: false } }
+              - column: {name: reference_type, type: nvarchar(64), constraints: { nullable: false } }
+
+        - addPrimaryKey:
+            constraintName: pk_${gravitee_prefix}page_acl
+            columnNames: page_id, reference_id, reference_type
+            tableName: ${gravitee_prefix}page_acl
+
+        - sql:
+            sql: insert into ${gravitee_prefix}page_acl (page_id, reference_id, reference_type) select page_id, excluded_group, 'GROUP' from ${gravitee_prefix}page_excluded_groups;
+        - sql:
+            sql: update ${gravitee_prefix}pages set excluded_access_controls = true, visibility = 'PRIVATE' from ${gravitee_prefix}page_excluded_groups where ${gravitee_prefix}page_excluded_groups.page_id = ${gravitee_prefix}pages.id and (${gravitee_prefix}page_excluded_groups.excluded_group IS NOT NULL OR ${gravitee_prefix}page_excluded_groups.excluded_group != '');
```

Maybe we can use the `<>` operator since it follows the ISO standard ?

```suggestion
            sql: update ${gravitee_prefix}pages set excluded_access_controls = true, visibility = 'PRIVATE' from ${gravitee_prefix}page_excluded_groups where ${gravitee_prefix}page_excluded_groups.page_id = ${gravitee_prefix}pages.id and (${gravitee_prefix}page_excluded_groups.excluded_group IS NOT NULL OR ${gravitee_prefix}page_excluded_groups.excluded_group <> '');
```